python-package/rgf/utils.py

```diff
@@ -559,6 +559,7 @@ def fit(self, X, y, sample_weight=None):
             raise ValueError("Classifier can't predict when only one class is present.")
 
         self._fitted = True
+        self.n_features_in_ = self._n_features
```

Comment:
Why not property: because now `scikit-learn` is checking only attributes with trailing underscore, not properties https://github.com/microsoft/LightGBM/issues/3014#issuecomment-678278744